RevenueCatUI/CustomerCenter/Views/ErrorView.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  ErrorView.swift
+//
+//  Created by Cesar de la Vega on 11/12/24.
+
+import SwiftUI
+
+#if os(iOS)
+
+@available(iOS 15.0, *)
+@available(macOS, unavailable)
+@available(tvOS, unavailable)
+@available(watchOS, unavailable)
+struct ErrorView: View {
+
+    @Environment(\.locale)
+    private var locale
+
+    var body: some View {
+        VStack(spacing: 20) {
+            let errorMessage: String = Localization.localizedBundle(self.locale)
+                .localizedString(forKey: "Something went wrong",
+                                 value: "Something went wrong",
+                                 table: nil)
+            CompatibilityContentUnavailableView(
```

Comment:
Hmm wondering if it would make sense to have a "retry" button... It might also be worth at least displaying network errors with a different message... But we can improve that in future PRs.

Reply:
I wanted to do that, but it involves checking the type of network error, and thought it would complicate things a bit. Let's leave it for a future PR. I will add it to the list of things to do